Sources/SwiftParser/SwiftParser.docc/FilingBugReports.md

```diff
@@ -20,7 +20,7 @@ If you have source code that is parsed correctly by the current Swift compiler (
 
 1. Run the following command to see the diagnostics produced by the parser
     ```
-    swift-parser-test print-diags /path/to/file.swift
+    swift-parser-cli print-diags /path/to/file.swift
```

Comment:
`print-diags` and `dump-tree` weren't updated in the docs. Personally I'd just prefer both be called `print-*` since `diags` and `tree` don't mean all that much by themselves. I'm not super attached to any of the names though.

Reply:
Good catch. I couldn’t decide between `print` and `dump`, so I removed it altogether. But thinking over it, I think the subcommand should contain a verb, so I changed it to `print-diags` and `print-tree`.

Reply:
Sounds good, thanks 🙇